src/github_runner_manager/openstack_cloud/openstack_runner_manager.py

```diff
@@ -202,16 +202,20 @@ def create_runner(self, registration_token: str) -> InstanceId:
         except OpenStackError as err:
             raise RunnerCreateError(f"Failed to create {instance_name} openstack runner") from err
 
+        logger.info("Waiting for runner process to startup: %s", instance.server_name)
```

Comment:
Could we use debug instead of info (applies to most of the logs introduced here)? It seems that these logs are mostly useful for troubleshooting and not for "normal" operation.

Reply:
Changed all the log levels to debug, thanks!